.swiftpm/xcode/xcshareddata/xcschemes/KSCrash-Package.xcscheme

```diff
@@ -105,6 +105,20 @@
                ReferencedContainer = "container:">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "BootTimeMonitor"
+               BuildableName = "BootTimeMonitor"
+               BlueprintName = "BootTimeMonitor"
+               ReferencedContainer = "container:">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
Is that needed?

Reply:
I don't know, I think it's automatic scheming, but I'm not sure. It did however change. Should I instead add it to the ignore list?

Reply:
reverted those changes.